pocs/utils/messaging.py

```diff
@@ -32,9 +32,9 @@ class PanMessaging(object):
     an instance of PanMessaging, on which they can then call
     receive_message.
 
-    Messages are sent to channels, a name that can be used to allow
-    a high-level partitioning of messages. A channel name may not
-    include whitespace. Among the currently used channel names are:
+    Messages are sent to topics, a name that can be used to allow
```

Comment:
Is this the best place to record all the current topics?

Reply:
My plan is to develop a scheme for future topics and to also write general documentation about topics and filtering. I've created issue #606 for that.